src/core/packages/saved-objects/import-export-server-internal/src/import/import_saved_objects.test.ts

```diff
@@ -554,6 +577,45 @@ describe('#importSavedObjectsFromStream', () => {
         expect(mockCreateSavedObjects).toHaveBeenCalledWith(createSavedObjectsParams);
       });
     });
+
+    describe('access control', () => {
+      test(`calls 'collectSavedObjects' with createAccessControlImportTransforms function if provided`, async () => {
+        const options = setupOptions({ accessControl: true });
+        const supportedTypes = ['foo-type'];
+        typeRegistry.getImportableAndExportableTypes.mockReturnValue(
+          supportedTypes.map((name) => ({ name })) as SavedObjectsType[]
+        );
+
+        await importSavedObjectsFromStream(options);
+        expect(typeRegistry.getImportableAndExportableTypes).toHaveBeenCalled();
+        const collectSavedObjectsOptions = {
+          readStream,
+          objectLimit,
+          supportedTypes,
+          typeRegistry,
+          createAccessControlImportTransforms,
```

Comment:
Alternative implementation is to handle execution of the transform conditionally, depending on the feature flag.
That way, we can always pass in `createAccessControlImportTransforms` and it will be a passthrough if not enabled.